MobileCenterDistribute/MobileCenterDistribute/MSDistributeUtil.m

```diff
@@ -39,3 +39,17 @@
     return stringToken;
   }
 }
+
+@implementation MSDistributeUtil
+
++ (BOOL)isRunningInDebugConfiguration {
```

Comment:
No. The method on Core checks for the debugger to be attached, we are not checking the debug config. I thought about moving this to Core but didn't. 
I'm moving it there, looking at the code now, it is odd for this to be in MSDistributeUtil.